src/index.tsx

```diff
@@ -8,22 +8,13 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import "./assets/main.scss";
 import "./App.css";
 import store from "./redux";
-
-let App;
```

Comment:
I think we still need this. From what I understand, the usecase is to deploy standalone Hub. @imsergiy, could you confirm?